readme.md

```diff
@@ -26,19 +38,90 @@ isAbsoluteUrl('foo/bar');
 ```
 
 
-## Related
+## Options Parameters  
 
-See [is-relative-url](https://github.com/sindresorhus/is-relative-url) for the inverse.
+### httpOnly  
+
```

Comment:
I would keep the docs simple like how it's done in the Usage section. A simple js block with lines and their result as comments would be better for readability.

Reply:
Understood. Can you provide feedback now please?